base-hack/src/fixes/dk_free_softlock.c

```diff
@@ -64,7 +63,7 @@ void cutsceneDKCode(void) {
 	}
 	// Define jump as having an animation timer > 80.0f
```

Comment:
Comment says what?

Reply:
Effectively the same. Just minor code opt to reduce casting